Makefile

```diff
@@ -83,8 +83,10 @@ help: ## Display this help.
 ##@ Development
 
 .PHONY: manifests
-manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
+manifests: controller-gen kustomize## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="{./api/...,./cmd/...,./controllers/...,./sidecar/...}" output:crd:artifacts:config=config/crd/bases
+	cd config/manager && $(KUSTOMIZE) edit set image controller=${CONTROLLER_IMG} $(KUSTOMIZE_RBAC_PROXY)
+	$(KUSTOMIZE) build config/default > deploy/controller/setup-controller.yaml
```

Comment:
With this, yaml will be auto-generated whenever manifests is run and can be verified in the ci https://github.com/csi-addons/kubernetes-csi-addons/runs/4977203144?check_suite_focus=true#step:4:10